jscomp/others/js_global.ml

```diff
@@ -0,0 +1,34 @@
+type intervalId
+type timeoutId
+(* part of WindowOrWorkerGlobalScope, to be implemented later
+type cacheStorage (** Service Workers, not yet implemented *)
+type idbFactory (** Indexed Database API, not yet implemented *)
+type imageBitmap (** not yet implemented *)
+type ('a, 'e) promise (* should this type be defined in Js, and be aliased by bs-promise and other implementations? *)
+*)
+
+(* part of WindowOrWorkerGlobalScope, to be implemented later
+external caches : cacheStorage = "" [@@bs.val] (* Service Workers, experimental *)
+external indexedDB : idbFactory = "" [@@bs.val] (* Indexed Database API *)
+external isSecureContext : bool = "" [@@bs.val] (* Secure Contexts, experimental *)
+external origin : string = "" [@@bs.val] (* WHATWG HTML Living Standard, experimental *)
+*)
+
+(** convert an ASCII-encoded string to its Base64-encoded equivalent *)
+external atob : string -> string = "" [@@bs.val]
+
+(** convert a Base64-encoded string to its ASCII-encoded equivalent *)
+external btoa : string -> string = "" [@@bs.val]
+
+external clearInterval : intervalId -> unit = "" [@@bs.val]
```

Comment:
They're a web standard, since HTML5, but not in the EcmaScript standard. See https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/atob

I suppose the line should be drawn where EcmaScript stops, so I'll remove them.